json11.cpp

```diff
@@ -321,6 +323,23 @@ bool Json::operator< (const Json &other) const {
  * Parsing
  */
 
+/* strtod_dot(str)
+ *
+ * Parse the number according to the JSON grammar, ignoring the current locale assuming that the
+ * string is already validated to comply with the JSON grammar.
+ */
+double strtod_dot(const char* str) {
```

Comment:
This seems like a good approach, but I don't see it called anywhere.  You probably meant to call it in parse_number().